manifests/init.pp

```diff
@@ -27,11 +27,11 @@
     fail('You cannot use the cd4pe module to install on EL 8')
   }
 
-  $compiling_server_osfamily = cd4pe::compiling_server_osfamily()
+  $compiling_server_osname = cd4pe::compiling_server_osname()
```

Comment:
If we are no longer using `compiling_server_osfamily`, then could you also please delete that file as it's only usage was here.

/cc @abottchen do you recall why you chose osfamily instead of osname? Is that what the PE modules do?

Reply:
Right, I essentially used the same logic as the puppet_enterprise module uses to determine whether or not the OS matches the PE master when installing a compiler.  That code uses `os['family']`: https://github.com/puppetlabs/puppet-enterprise-modules/blob/master/modules/puppet_enterprise/lib/shared/aio_build.rb#L4.

That said, it seems reasonable to use os['name'] if that produces a more precise string that improves the error message text.  It is still a core fact that we should be able to rely on.